README.md

```diff
@@ -36,7 +36,7 @@ First, the HTML fragment for the inputs and a place to display the live result:
 Using `@most/core` to make it reactive:
 
 ```es6
-import { combine, newDefaultScheduler } from '@most/core'
+import { combine, runEffects, newDefaultScheduler } from '@most/core'
```

Comment:
I had gone with the naming from #21, but it probably makes sense to stick with the naming from master until #21 is merged.  I'll change it, thanks!